### PR TITLE
feat: add radio button plugin

### PR DESCRIPTION
**Pull request by licam**

### CODE REVIEW

File: radio-button-plugin/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.0.1",
+  "description": "Sample radio plugin for inquirer-gui",
+  "main": "dist/radioPlugin.umd.js",
+  "scripts": {
+    "build": "vue-cli-service build --target lib --name radioPlugin src/index.js",
+    "prepublishOnly": "npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/inquirer-gui/sample-plugin"
```
Comment (alex-gilin) [inline radio-button-plugin/package.json:12]: url is incorrect

File: radio-button-plugin/src/packages/RadioGroup.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<script>
+export default {
+  name: "RadioGroup",
+  props: {
+    question: Object
+  },
+  methods: {
+    getRadioId(name){
+        return "radio_"+ name;
```
Comment (alex-gilin) [inline radio-button-plugin/src/packages/RadioGroup.vue:23]:
```suggestion
        return `radio_${name}`;
```

File: radio-button-plugin/src/packages/RadioGroup.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    onClick(answer) {
+      if (answer !== undefined) {
```
Comment (alex-gilin) [inline radio-button-plugin/src/packages/RadioGroup.vue:26]:
```suggestion
      if (answer) {
```

File: radio-button-plugin/src/packages/RadioGroup.vue [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        this.$emit("answerChanged", this.question.name, answer);
+      }
+    },
+    isVisible() {
+      return this.hasItem() && (this.question.visible!==false);
```
Comment (alex-gilin) [inline radio-button-plugin/src/packages/RadioGroup.vue:31]:
```suggestion
      return this.hasItem() && this.question.visible;
```

File: sample-app/src/App.vue [hunk 5]
```diff
@@ -29,6 +29,8 @@ import main from "./main.js";
 import DatePlugin from "@sap-devx/inquirer-gui-date-plugin";
 import LabelPlugin from "@sap-devx/inquirer-gui-label-plugin";
 import FileBrowserPlugin from "@sap-devx/inquirer-gui-file-browser-plugin";
+//import RadioGroup from "@sap-devx/inquirer-gui-radio-button-plugin";
+import RadioGroup from "../../radio-button-plugin/src/packages/RadioGroup";
```
Comment (alex-gilin) [inline sample-app/src/App.vue:33]: seems like `import` from the commented line should be used here

**Review by alex-gilin — CHANGES_REQUESTED** [review]
- `sample-app` in browser looks ugly in browser (seems like css styles is missed there)
- `simple-vscode-extension` needs adjustment for radio-button

File: radio-button-plugin/package.json (hunk 1 above)
Comment (vevush) [reply radio-button-plugin/package.json]: changed

File: radio-button-plugin/src/packages/RadioGroup.vue (hunk 2 above)
Comment (vevush) [reply radio-button-plugin/src/packages/RadioGroup.vue]: changed

File: radio-button-plugin/src/packages/RadioGroup.vue (hunk 3 above)
Comment (vevush) [reply radio-button-plugin/src/packages/RadioGroup.vue]: changed

File: radio-button-plugin/src/packages/RadioGroup.vue (hunk 4 above)
Comment (vevush) [reply radio-button-plugin/src/packages/RadioGroup.vue:31]: this change doesn't work

File: sample-app/src/App.vue (hunk 5 above)
Comment (vevush) [reply sample-app/src/App.vue]: changed

File: sample-app/src/App.vue [hunk 6]
```diff
@@ -29,6 +29,8 @@ import main from "./main.js";
 import DatePlugin from "@sap-devx/inquirer-gui-date-plugin";
 import LabelPlugin from "@sap-devx/inquirer-gui-label-plugin";
 import FileBrowserPlugin from "@sap-devx/inquirer-gui-file-browser-plugin";
+import RadioGroup from "@sap-devx/inquirer-gui-radio-plugin";
+//import RadioGroup from "../../radio-button-plugin/src/packages/RadioGroup";
```
Comment (alex-gilin) [inline sample-app/src/App.vue:33]: consider to remove commented import row

**Review by alex-gilin — CHANGES_REQUESTED** [review]

File: sample-app/src/App.vue (hunk 6 above)
Comment (vevush) [reply sample-app/src/App.vue]: removed

**Review by alex-gilin — APPROVED** [review]

**Pull request merged**